frame/oracle/src/lib.rs

```diff
@@ -373,6 +375,7 @@ pub mod pallet {
 			ensure!(max_answers >= min_answers, Error::<T>::MaxAnswersLessThanMinAnswers);
 			ensure!(threshold < Percent::from_percent(100), Error::<T>::ExceedThreshold);
 			ensure!(max_answers <= T::MaxAnswerBound::get(), Error::<T>::ExceedMaxAnswers);
+			ensure!(block_interval > T::StalePrice::get(), Error::<T>::BlockIntervalLength);
```

Comment:
may be rename StalePrice -> BlockIntervalPriceOutdated?

Reply:
So stale price is how long before we remove prices if they are are not aggregated, this check is to make sure that, that is done before a re aggregation